master/pkg/model/model.go

```diff
@@ -4,36 +4,39 @@ import (
 	"fmt"
 	"strings"
 	"time"
+
+	"github.com/google/uuid"
 )
 
 // Model represents a row from the `models` table.
 type Model struct {
```

Comment:
I see that this is unused except for tests. Can we just move these models to a test file (maybe like api_checkpoint_intg_test.go) rather than like changing this code even though it is never used?

Reply:
I don't follow. Are you suggesting we maintain an outdated copy of Model struct for tests, and an updated one for queries?

Reply:
I guess my suggestion is that given ```model.Model``` was unused before this change except for tests which really don't use the model efficiently, I don't think this model is worth having and we should delete it or move it to be test only. 

I'd much rather delete the model now so we don't risk it being out of date or becoming out of date. Someone could still add it back when a change will actually use it beyond getting a table name.

Reply:
The query in `workspaceHasModels` requires an accurate `model.Model`:
```	
exists, err := db.Bun().NewSelect().Model((*model.Model)(nil)).
	Where("workspace_id=?", workspaceID).
	Exists(ctx)
```

I can delete `model.ModelVersion` though if you prefer.

Reply:
Exists queries don't require a model, you can instead give it a table name

```
exists, err := db.Bun().NewSelect().Table("models").
    Where("workspace_id = ?", workspaceID).
    Exists(ctx)
```

I'm also fine if you want to leave the model.

Reply:
I tried moving it and got stuck in a few import loops. I think there's probably a way to accomplish it but I'm not seeing it and I've spent as much time as I am willing to at this point. 😞

Reply:
oh yeah sorry I didn't realize it was used in ```internal/db``` tests

internal imports from db so if you put it in ```internal/db/postgres_test_utils.go``` you should be good